pkg/discovery/agent.go

```diff
@@ -132,7 +133,7 @@ func (a *Agent) Stop() {
 // discovery loop.
 func validateAPI() func(apiID, stageName string) bool {
```

Comment:
I'm not sure the value of this function since we do not rely on the cache from the sdk, since it only handles api services and not revisions.